lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -765,10 +774,10 @@ void OCLBackend::doForwardPass() {
 
```

Comment:
Isn't this a dead code branch? Because we replace Conv with OCLConvolution? If yes, then I suggest to delete this codepath first, maybe in separate PR.

Reply:
It is not dead. It is still used for training.

Reply:
I understand that ConvolutionGradInst is still used.
But forward ConvolutionNode is converted to OCLConvolutionNode, and then to OCLConvolutionInst, isn't it?

Reply:
No, ConvolutionNode is not replaced by OCLConvolutionNode in the the training mode. There are some issues with this, so this transformation is disabled in this case.

Reply:
yeah, i was looking at the same recently: https://github.com/pytorch/glow/blob/master/lib/Backends/OpenCL/Transforms.cpp#L53